packages/webapp/src/apiConfig.js

```diff
@@ -37,33 +37,31 @@ export const userUrl = `${URI}/user`;
 export const pseudoUserUrl = `${URI}/user/pseudo`;
```

Comment:
Thank you for working on this! Everything looks good to me.
Were you going to remove comments on this file?